#161714461 create an endpoint to get all of a user's followers

ucheg6 opened the pull request:
#### What does this PR do?
- create an endpoint to get all of a user's followers
#### Description of Task to be completed?
- write a function to get all of a user's followers
- test all functionalities
#### How should this be manually tested?
- Git clone the repository
- cd into the `haven-ah-backend` folder
- run `npm run start-dev`
- Fire up the route `localhost:5000/api/v1/profiles/user/followers` as a get request
- To test the entire application, run `npm run test`
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[#161714461](https://www.pivotaltracker.com/story/show/161714461)
#### Screenshots (if appropriate)
<img width="925" alt="screen shot 2018-11-15 at 12 30 50 pm" src="https://user-images.githubusercontent.com/34472497/48557278-9c778200-e8e6-11e8-9cff-e6dca62cbb7c.png">
<img width="1061" alt="screen shot 2018-11-15 at 2 58 03 pm" src="https://user-images.githubusercontent.com/34472497/48557394-ea8c8580-e8e6-11e8-8059-dac5e6fb1c33.png">

TheoOkafor reviewed (commented):
It all looks good. Nice work @ucheg6

TheoOkafor reviewed (approved):
It all looks good. Nice work @ucheg6

ucheg6 committed:
feat(followers): create an endpoint to get all of a user's followers

- write a function to get all of a user's followers
- test all functionalities

[finishes #161714461]
